### PR TITLE
Path

### PR DESCRIPTION
**Pull request by nikgraf**
Fixes https://github.com/carteb/carte-blanche/issues/286

### CODE REVIEW

File: plugins/react/frontend/components/common/IFrame/index.js [hunk 1]
```diff
@@ -4,8 +4,18 @@
...
-  `<!DOCTYPE html>
+const createHtml = (
+  componentPath,
+  publicPath = '',
+  dest = '',
+  userFiles,
+  injectTags,
+  commonsChunkFilename
+) => {
+  const basePath = publicPath || dest ? path.join('/', publicPath, dest) : '';
```
Comment (mxstbr) [inline plugins/react/frontend/components/common/IFrame/index.js]: My idea was to move this to the webpack plugin and pass that down to the React plugin?

Comment (nikgraf) [inline plugins/react/frontend/components/common/IFrame/index.js]: Yep, that's better. I will do that

**Comment by nikgraf** [comment]
@mxstbr can you review it?

File: webpack-plugin/src/apply.js [hunk 2]
```diff
@@ -14,8 +14,9 @@ import readMultipleFiles from 'read-multiple-files';
 import emitAssets from './utils/emitAssets';
 import registerPlugins from './registerPlugins';
 import registerDefaultPlugins from './registerDefaultPlugins';
-import createHTML from './utils/createHTML';
+import createHtml from './utils/createHtml';
```
Comment (mxstbr) [inline webpack-plugin/src/apply.js]: Why camel case it? That's sooo ugly

**Comment by mxstbr** [comment]
This will also allows us to make #209 happen! :grin:

File: webpack-plugin/src/apply.js [hunk 3]
```diff
@@ -40,7 +41,7 @@ function apply(compiler) {
   }
 
   // Compile the client
-  const userBundleFileName = path.join(dest, 'user-bundle.js');
+  const userBundleFileName = path.join(this.options.dest, 'user-bundle.js');
```
Comment (mxstbr) [inline webpack-plugin/src/apply.js]: Shouldn't this be `path.join(basePath, 'user-bundle.js')`?

**Comment by mxstbr** [comment]
Looks good, two small nitpicks!

File: webpack-plugin/src/apply.js (hunk 3 above)
Comment (nikgraf) [inline webpack-plugin/src/apply.js]: Nope, this is used by webpack and the publicPath is prepended by default here (at least when I tested it)

**Comment by nikgraf** [comment]
I just merged so I can do more things while driving home to Vienna.

**Pull request merged**